include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -416,6 +416,50 @@ __get_accessor_ptr(const _Acc& __acc)
 #endif
 }
 
+class __event
+{
+    sycl::event __m_event;
+    sycl::queue __m_queue;
+    bool __in_order = false;
+
+  public:
+    __event(sycl::event __e): __m_event(__e) {}
+    __event() {}
```

Comment:
`__event() = default;`

Reply:
What's a advantages of that in given case?

Reply:
In common case `=default` doesn't change triviality state of class.
See https://ru.stackoverflow.com/questions/495342/%D0%A7%D0%B5%D0%BC-%D0%BF%D1%83%D1%81%D1%82%D0%BE%D0%B9-%D0%BA%D0%BE%D0%BD%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D0%BE%D1%80-%D0%BF%D0%BE-%D1%83%D0%BC%D0%BE%D0%BB%D1%87%D0%B0%D0%BD%D0%B8%D1%8E-%D0%BE%D1%82%D0%BB%D0%B8%D1%87%D0%B0%D0%B5%D1%82%D1%81%D1%8F-%D0%BE%D1%82-default for details.

Reply:
"doesn't change triviality state of class." - yes...
But, do we really need it? or it can be useful in the Future?

Reply:
This general idea look like as applied in all cases.